src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -78,6 +78,29 @@ public int SelectedIndex
             }
         }
 
+        public string getNameOfSelectedDropDownItem ()
+        {
+            return (items.ElementAt(selectedIndex)).Name;
+        }
+
+        public object getItemOfSelectedDropDownItem()
+        {
+            return (items.ElementAt(selectedIndex)).Item;
+        }
+
+        public int getIndexOfItem(string itemName)
```

Comment:
This can be rewritten with Linq as follow:

```csharp
private int GetIndexOfItem(string itemName)
{
    if (string.IsNullOrEmpty(itemName)) return -1;

    return items.ToList().FindIndex(item => item.Name.Equals(itemName));
}
```